### PR TITLE
Update prometheus_exporter.py

### PR DESCRIPTION
**Pull request by tyun08**
Remove task_status metrics monitoring

### CODE REVIEW

File: airflow_exporter/prometheus_exporter.py [hunk 1]
```diff
@@ -103,19 +103,21 @@ def collect(self):
...
+
+#             t_state = GaugeMetricFamily(
+#                 'airflow_task_status',
+#                 'Shows the number of task starts with this status',
+#                 labels=['dag_id', 'task_id', 'owner', 'status', 'hostname'] + k
+#             )
+#             for task in tasks:
+#                 t_state.add_metric([task.dag_id, task.task_id, task.owners, task.state or 'none', task.hostname or 'none'] + v, task.value)
             
-            yield t_state
```
Comment (mijailg) [inline airflow_exporter/prometheus_exporter.py]: You can just delete code. Thats why we have versioning.

**Review by mijailg — COMMENTED** [review]
How do you plan on hosting the library? I don't think we have a pypi server.

File: airflow_exporter/prometheus_exporter.py (hunk 1 above)
Comment (tyun08) [reply airflow_exporter/prometheus_exporter.py]: Removed.

**Comment by tyun08** [comment]
> How do you plan on hosting the library? I don't think we have a pypi server.

The easiest way I think is to update the pip requirements.txt file to point the package to this repo, like:
-e git@github.com:CapsuleHealth/airflow-exporter.git

and remove the:
airflow-exporter==1.3.0

But this way we have to redeploy the image. Otherwise we can manually install the repo on all nodes?

**Comment by mijailg** [comment]
I think git might work. Here is the base image for Airflow: https://github.com/CapsuleHealth/airflow-image.

**Pull request merged**